rectools/metrics/ranking.py

```diff
@@ -37,8 +40,12 @@ class _RankingMetric(MetricAtK):
     ----------
     k : int
         Number of items at the top of recommendations list that will be used to calculate metric.
+    debias_config : DebiasConfig, default None
+        Config with debias method parameters (iqr_coef, random_state).
     """
 
+    debias_config: DebiasConfig = attr.ib(default=None)
```

Comment:
Probably we can add this to `MetricAtK`? Together with the code for handling it

Reply:
Debias is not applicable to all of the metrics. I suggest to add it only to appicable types of metrics.

Reply:
Let's  then probably add a class like `DibiasableMetrikAtK`?

Reply:
Let's  then probably add a class like `DibiasableMetrikAtK`?

Reply:
I don't think it will solve this point, as there will also be classification and ranking metrics inherited from it, which can be and are used without the debias mechanism

Reply:
@In48semenov what are the examples? I think we can probably debias anything that belongs to classification and ranking

Reply:
Sorry, I misunderstood the previous comments

Yes, `DibiasableMetrikAtK` class would look quite reasonable, as it would be easy to add new debias methods to it in the future

Reply:
I added `DibiasableMetrikAtK` to `debias.py`